plugins/woocommerce-admin/client/marketing/data-multichannel/resolvers.ts

```diff
@@ -47,3 +55,48 @@ export function* getRecommendedChannels() {
 		throw error;
 	}
 }
+
+const getTotalFromResponse = ( response: Response ) => {
```

Comment:
💅  Missed JSDOCS

Reply:
Done in [`b838885` (#36735)](https://github.com/woocommerce/woocommerce/pull/36735/commits/b83888577a6b80889f7c8126875ac28e702219df).